core/cas-server-core-configuration/src/main/java/org/apereo/cas/configuration/model/support/ldap/LdapAuthenticationProperties.java

```diff
@@ -51,6 +51,8 @@
      */
     private String principalAttributeId;
 
+    private String principalDnAttributeName;
```

Comment:
This needs to be documented explicitly and clearly, like all other settings. It also needs to include a default value.

Reply:
Ok, I will fix it right now.
Btw, I see that only allowMissingPrincipalAttributeValue has a default value in this same file...

Reply:
Yup. That's good. The rest likely need to move over from the actual handler at some point.